Content.Server/_RMC14/Admin/RMCRecalculatePower.cs

```diff
@@ -0,0 +1,19 @@
+using Content.Server.Administration;
+using Content.Shared._RMC14.Power;
+using Content.Shared.Administration;
+using Robust.Shared.Console;
+
+namespace Content.Server._RMC14.Admin;
+
+[AdminCommand(AdminFlags.Admin)]
```

Comment:
Needs to be Debug perm as well

Reply:
`[AdminCommand(AdminFlags.Admin), AdminCommand(AdminFlags.Debug)]`

or

`[AdminCommand(AdminFlags.Admin | AdminFlags.Debug)]`

Reply:
Just debug